program-runtime/src/compute_budget.rs

```diff
@@ -73,13 +73,8 @@ impl Default for ComputeBudget {
 
 impl ComputeBudget {
     pub fn new(use_max_units_default: bool) -> Self {
-        let max_units = if use_max_units_default {
-            MAX_UNITS
-        } else {
-            200_000
-        } as u64;
         ComputeBudget {
-            max_units,
```

Comment:
Can we change this such that the `ComputeBudget` just passes in the `max_units` instead of `use_max_units_default`. 

Then the constructor of `ComputeBudget` can either use one of:
1. `ComputeBudget::new(DEFAULT_UNITS)` (constant suggested by this comment here: https://github.com/solana-labs/solana/pull/23656/files#r827652115)
2. `ComputeBudget::new(MAX_UNITS)`